sims/verilator/Makefile

```diff
@@ -1,74 +1,104 @@
-#########################################################################################
+#############################################################################
+# Extra verilator-specific Makefile variables:
+#
+# ENABLE_PRINTF_PATTERN - python regex. any printf() with text that matches
+#                         this regex will be force enabled in sim output
+#                         default is no selectively enabled printf's
+# VERILATOR_PROFILE     - 'none' if no verilator profiling (default)
+#                         'all' if full verilator runtime profiling
+#                         'threads' if runtime thread profiling only
+# VERILATOR_THREADS     - how many threads the simulator will use (default 1)
+# VERILATOR_FST_MODE    - enable FST waveform instead of VCD. only makes sense
+#                         if debug simulator is built
+#############################################################################
+
+
+#############################################################################
 # verilator makefile
-#########################################################################################
```

Comment:
I don't understand why you went to the extra effort to shorten these lines...

Reply:
It displays somewhat better in narrower terminals with 80 columns.  I'm guessing that shortening it to 77 characters leaves room for a 2 or 3-digit line number in the text editor, although 72 would be the more conservative and conventional choice.

Reply:
That seems fine, but I still think the style changes should be a separate PR

Reply:
i'm going to leave comment-line length changes in this PR for the files i modified. This file's contents were rewritten to fit into 80 characters, so the comments would be the only lines in that file that wrap outside of 80 characters. This would be silly.